.github/buildomat/jobs/phd-build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#:
+#: name = "phd-build"
+#: variety = "basic"
+#: target = "helios"
+#: rust_toolchain = "stable"
+#: output_rules = [
+#:   "/out/*",
+#: ]
+#:
+#: [[publish]]
+#: series = "image"
+#: name = "propolis-server-debug.tar.gz"
+#: from_output = "/out/propolis-server-debug.tar.gz"
+#:
+#: [[publish]]
+#: series = "image"
+#: name = "propolis-server-debug.sha256.txt"
+#: from_output = "/out/propolis-server-debug.sha256.txt"
+#:
+#: [[publish]]
+#: series = "image"
+#: name = "phd-runner.tar.gz"
+#: from_output = "/out/phd-runner.tar.gz"
+#:
+#: [[publish]]
+#: series = "image"
+#: name = "phd-runner.sha256.txt"
+#: from_output = "/out/phd-runner.sha256.txt"
```

Comment:
Are you planning to have an external tool that is going to locate these files by the commit ID of the change that generated them, and download them?  If not, the `output_rules` stuff is sufficient for them to make their way into `/input`; publish is just for providing a stable external URL for tools outside this CI run.

Reply:
No, not at this stage (if ever). I've gone ahead and removed these with 628aba9.